src/components/VmDetails/cards/OverviewCard/style.css

```
@@ -46,3 +46,14 @@
   line-height: 20px;
   padding-left: 10px;
 }
+
+@media only screen and (max-width: 1024px) {
+  .media-left {
+    display: block;
+  }
+  
+  .media-body {
```

```suggestion
  :global(.media-body) {
```
Then you can drop the `className` attribute on the `Media.Body` tag.